app/javascript/components/retirement-form/index.jsx

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from 'react';
+
+import MiqFormRenderer from '@@ddf';
+import createSchema from './retirement-form.schema';
+import { API, http } from '../../http_api';
+import handleFailure from '../../helpers/handle-failure';
+
+const RetirementForm = ({ retirementID }) => {
+  console.log(retirementID)
+  const id = retirementID.split('"').filter(Number);
+  const [{ initialValues, isLoading }, setState] = useState({
+    isLoading: !!id,
+    initialValues: {
+      retirementDate: null,
+      retirementWarning: '',
+      formMode: '',
+    },
```

Comment:
Do we really need these? If yes, they should probably go to the schema...

Reply:
Only formMode was needed, I removed the rest.